gpkit/constraints/set.py

```diff
@@ -44,8 +38,11 @@ def __getitem__(self, key):
             if len(variables) == 1:
                 return variables[0]
             else:
-                variables.sort(key=_var_sort_key)
-                return variables
+                sortfn = lambda v: len(v.key.models) if v.key.models else 0
```

Comment:
http://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def